ui/components/src/EChart/EChart.tsx

```diff
@@ -182,7 +183,20 @@ export const EChart = React.memo(function EChart<T>({
     updateSize();
   }, [sx]);
 
-  return <Box ref={containerRef} sx={sx}></Box>;
+  return (
+    <Box
+      ref={containerRef}
+      sx={combineSx(
+        {
+          // Ensures chart fills container and updates size correctly when resizing,
+          // combineSx allows this default behavior to be overriden from parent component
```

Comment:
I might say "this behavior can be overridden with combineSx on child components" (make it more direct)

Reply:
Hmm agreed this comment is a little confusing, but EChart.tsx actually is the child component and combineSx is allowing the parent component (LineChart, StatChart, GaugeChart) an easy way to customize it using sx as an escape hatch (in case the new default width and height aren't ideal for some edge case)